app/views/metrics/show.html.erb

```diff
@@ -25,18 +25,18 @@
     <%= render "components/related-actions",
       {
         links: [
-                  {
-                    header: t(".navigation.visit_page"),
-                    link_url: "//www.gov.uk#{@performance_data.base_path}",
-                    accessibility_message: t('.navigation.accessibility_message'),
-                    label: @performance_data.title
-                  },
-                  {
-                    header: "Make changes to the page",
-                    link_url: "#",
-                    label: t('.navigation.edit_link', publisher_app: @performance_data.publishing_app)
-                  }
-                ]
+          {
+            header: t(".navigation.visit_page"),
+            link_url: "//www.gov.uk#{@performance_data.base_path}",
+            accessibility_message: t('.navigation.accessibility_message'),
+            label: @performance_data.title
+          },
+          {
+            header: t(".navigation.edit_page"),
+            link_url: @performance_data.edit_url,
+            label: @performance_data.edit_label
```

Comment:
Should we add the accessibilty message?

Reply:
I hadn't looked at that, but I've now had a look.

From what I can tell, this adds some useful context to the links to GOV.UK content (as otherwise it's just the title of the content), but for this link the text will either be "Request a change to mainstream content" or "Edit in #{app}", which seem OK without any additional clarification.